Code/Engine/RendererVulkan/Utils/Implementation/ImageCopyVulkan.cpp

```
@@ -493,6 +493,8 @@ void ezImageCopyVulkan::RenderInternal(const ezVec3U32& sourceOffset, const vk::
         {
           write.pImageInfo = &sourceInfo;
         }
+        default:
```

should this silently continue ?

---

Good point, that was just my attempt to silence clang. I've added asserts in both unused code paths now.